po/fr.po

```diff
@@ -221,354 +242,354 @@ msgstr "%ls (ligne %lu) : "
+msgstr "%ls : %ls %ls : ces options ne peuvent pas être utilisées ensembles\n"
 
 #, c-format
 msgid "%ls: %ls: cannot overwrite read-only variable"
-msgstr ""
+msgstr "%ls : %ls : impossible de surcharger une variable en lecture seule"
 
 #, c-format
 msgid "%ls: %ls: cannot use reserved keyword as function name"
-msgstr ""
+msgstr "%ls : %ls : impossible d’utiliser un mot-clé réservé comme nom de fonction"
 
 #, c-format
 msgid "%ls: %ls: contains a syntax error\n"
-msgstr ""
+msgstr "%ls : %ls : contient une erreur de syntaxe\n"
 
 #, c-format
 msgid "%ls: %ls: expected %d arguments; got %d\n"
-msgstr ""
+msgstr "%ls : %ls : %d arguments attendus ; %d reçus\n"
 
 #, c-format
 msgid "%ls: %ls: invalid base value\n"
-msgstr ""
+msgstr "%ls : %ls : valeur de base invalide\n"
 
 #, c-format
 msgid "%ls: %ls: invalid function name"
-msgstr ""
+msgstr "%ls : %ls : nom de fonction invalide"
 
 #, c-format
 msgid "%ls: %ls: invalid integer\n"
-msgstr ""
+msgstr "%ls : %ls : entier invalide\n"
 
 #, c-format
 msgid "%ls: %ls: invalid mode\n"
-msgstr ""
+msgstr "%ls : %ls : mode invalide\n"
 
 #, c-format
 msgid "%ls: %ls: invalid mode name. See `help identifiers`\n"
-msgstr ""
+msgstr "%ls : %ls : nom de mode invalide. Voir `help identifiers`\n"
 
 #, c-format
 msgid "%ls: %ls: invalid process id"
-msgstr ""
+msgstr "%ls : %ls : identifiant de processus invalide"
 
 #, c-format
 msgid "%ls: %ls: invalid scale\n"
-msgstr ""
+msgstr "%ls : %ls : échelle invalide\n"
 
 #, c-format
 msgid "%ls: %ls: invalid subcommand\n"
-msgstr ""
+msgstr "%ls : %ls : sous-commande invalide\n"
 
 #, c-format
 msgid "%ls: %ls: invalid variable name. See `help identifiers`\n"
-msgstr ""
+msgstr "%ls : %ls : nom de variable invalide. Voir `help identifiers`\n"
 
-#
 #, c-format
 msgid "%ls: %ls: option does not take an argument\n"
-msgstr ""
+msgstr "%ls : %ls : cette option ne prend pas d’argument\n"
 
 #, c-format
 msgid "%ls: %ls: option requires an argument\n"
-msgstr ""
+msgstr "%ls : %ls : cette option doit être utilisé avec un argument\n"
 
 #, c-format
 msgid "%ls: %ls: subcommand takes no options\n"
-msgstr ""
+msgstr "%ls : %ls : cette sous-command n’a pas d’option\n"
 
 #, c-format
 msgid "%ls: %ls: unexpected positional argument"
-msgstr ""
+msgstr "%ls : %ls : argument positionnel inattendu"
 
 #, c-format
 msgid "%ls: %ls: unknown option\n"
-msgstr ""
+msgstr "%ls : %ls : option inconnue\n"
 
 #, c-format
 msgid "%ls: '%ls' is a broken symbolic link to '%ls'\n"
-msgstr ""
+msgstr "%ls : « %ls » est un lien symbolique cassé vers « %ls »\n"
 
 #, c-format
 msgid "%ls: '%ls' is not a directory\n"
-msgstr "%ls : '%ls' n’est pas un dossier\n"
+msgstr "%ls : « %ls » n’est pas un dossier\n"
 
 #, c-format
 msgid "%ls: '%ls' is not a job\n"
-msgstr "%ls : '%ls' n’est pas une tâche\n"
+msgstr "%ls : « %ls » n’est pas une tâche\n"
 
 #, c-format
 msgid "%ls: '%ls' is not a valid job id\n"
-msgstr ""
+msgstr "%ls : « %ls » n’est pas un identifiant de tâche valide\n"
```

Comment:
In this case, "id" should be spelled "ID" if you want to change that as well (at the source).